internal/provider/pipeline_resource.go

```diff
@@ -112,6 +121,18 @@ For more details, take a look at: https://docs.warpstream.com/warpstream/configu
 				Description: "The unique identifier of the pipeline configuration, automatically generated by WarpStream upon creation.",
 				Computed:    true,
 			},
+			"type": schema.StringAttribute{
+				Description: "Pipeline type. Valid types are: `bento` (default) and `orbit`",
+				Optional:    true,
+				Computed:    true,
+				Default:     stringdefault.StaticString(bentoPipelineType),
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+				Validators: []validator.String{
+					stringvalidator.OneOf([]string{bentoPipelineType, orbitPipelineType}...),
```

Comment:
```suggestion
					stringvalidator.OneOf(bentoPipelineType, orbitPipelineType),
```

I see we do the `[]string{}...` thing in a few places but I think that's just an oversight.